app/controllers/smart_proxies_controller.rb

```diff
@@ -85,6 +85,40 @@ def destroy
     end
   end
 
+  def log_pane
+    render :partial => 'smart_proxies/logs/list', :locals => {:log_entries => @smart_proxy.statuses[:logs].log_entries}
+  rescue Exception => e
```

Comment:
Rescue any Exception? I'd say just rescue [StandardError instead](https://robots.thoughtbot.com/rescue-standarderror-not-exception), but if you could compile a list of possible exceptions that'd be much better. I realize you're showing the error in process_ajax_error, but that can inadvertently hide long backtraces.